example/webpack.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
     extensions: [".js", ".ts", ".jsx", ".tsx"],
     alias: {
       "@": path.resolve(__dirname, "assets/js"),
-      // Necessary to deduplicate React due to npm link:
```

Comment:
Maybe remove mention to link, as it isn't a link anymore?

Reply:
I left the mention because we use `link` when we are testing local changes made to the client